core/src/main/java/com/amplifyframework/core/model/ModelSchema.java

```diff
@@ -78,6 +80,9 @@ public final class ModelSchema {
     // Class of the model this schema will represent
     private final Class<? extends Model> modelClass;
 
+    // Model schema version
+    private final int modelSchemaVersion;
```

Comment:
Please update the implementation of `ModelSchema#equals` to compare the newly added properties.

Reply:
Resolved